EIPS/eip-969.md

```diff
@@ -0,0 +1,167 @@
+inherent security risks involved with changing the mining algorithm with the
+risk that the change we make does not break existing ASIC miners.  This EIP
+leans towards minimizing the security risks by making minimal changes to the
+algorithm accepting the risk that the change may not break dedicated hardware 
+miners that utilize partially or fully configurable logic. 
+
+Furthermore, we do not wish to introduce significant algorithm changes that
+may alter the power utilization or performance profile of existing GPU hardware.
+
+The change of FNV constant is a minimal change that can be quickly
+implemented across the various network node and miner implementations.
+
+It is proposed that `ASIC_MITIGATION_FORK_BLKNUM` be no more than 5550000 (epoch 185), giving
+around 30 days of notice to node and miner developers and a sufficient window
+for formal analysis of the changes by experts. We must weigh this window against
+the risk introduced by allowing ASICs that may exist to continue to propagate
+on the network, as well as the risk of providing too much advanced warning to 
+ASIC developers. 
+
+It is further understood that this change will not prevent redesign of existing
+dedicated hardware with new ASIC chips. The intention of this change is only
+to disable currently active or mid-production hardware and provide time for
+POS development as well as larger algorithim changes to be well analyzed by 
+experts.
+
+The choice of FNV constants is made based on the formal specification at
+https://tools.ietf.org/html/draft-eastlake-fnv-14#section-2.1
+
+Typical ASIC synthesis tools would optimize multiplication of a constant
+in the FNV algorithm, reducing the area needed for the multiplier according
+to the hamming weight of the constant. To reduce the chance of ASIC adaptation
+through minor mask changes, we propose choosing new constants with a larger
+hamming weight, however care should be taken not to choose constants with too
+large of weight.
+
+The current FNV prime, 0x1000193 has a hamming weight of 6.   
+`HammingWeight(0x10001a7) = 7;`  
+`HammingWeight(0x10001ab) = 7;`    
+`HammingWeight(0x10001cf) = 8;`    
+`HammingWeight(0x10001e3) = 7;`    
+`HammingWeight(0x10001ef) = 9; // Not chosen`   
+`HammingWeight(0x10001f9) = 8;`  
+`HammingWeight(0x10001fb) = 9; // Not chosen`
+
+An exhaustive analysis was done regarding the dispersion of these constants as compared to 0x01000193.
+
+It can be empirically confirmed that no more than 1 duplicates occur in the 32 bit word space with these constants. 
+
+It is worth noting that FNV is not a cryptographic hash, and it is not used as such in ethash. With 
+that said, a more invasive hash algorithm change could consider other options. One suggestion has been 
+MurmorHash3 (https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp)
+
+## Backwards Compatibility
+
+This change implements a backwards incompatable change to proof of work based
```

Comment:
incompatable -> incompatible

Reply:
This is what I get for using vi without spell checker. Grammar fixes incoming.